configure.py

```diff
@@ -2175,4 +2176,7 @@ def make_bin_override():
 
 print_verbose("running: \n    " + " ".join(['python', 'tools/gyp_node.py'] + gyp_args))
 run_gyp(gyp_args)
+if sys.platform == 'win32':
```

Comment:
```suggestion
if options.compile_commands_json and sys.platform == 'win32':
```

If I'm not mistaken, this should only run if `compile_commands_json` is set.